opm/input/eclipse/EclipseState/Tables/TableSchema.hpp

```diff
@@ -30,6 +30,13 @@ namespace Opm {
 
     class TableSchema {
     public:
+        TableSchema() = default;
+
+        TableSchema(const OrderedMap<std::string,ColumnSchema>& columns)
+          : m_columns(columns)
+        {
+        }
```

Comment:
Just out of curiosity, why do we need these constructors?

Reply:
woops, we don't, not supposed to be in here.